### PR TITLE
docs: documented rollback strategy

### LINKED ISSUE #360: Create Rollback branch 🌳
### Expected Behavior

Now that we have finished the development of all v1 features we should have a simple rollback strategy.

A snapshot branch should track the current state of V1 and the proper procedure to rollback to that state should be documented for the core team.

### Current Behavior

Our only stable version of the code lives on dev at the moment 😬 😰

### Any other notes

_No response_

### Assignment

This issue is free for anyone to take

### PR DESCRIPTION
**Pull request by camelPhonso**
# Description

**Closes #360**  

This PR is only for the changes to documentation that explain the current rollback plan. A snapshot branch has already been setup with the currently stable version of the platform and can be found by following the documentation as per the changes committed here.

### Files changed

- `CONTRIBUTING.md`

### UI changes

n/a

### Changes to Documentation

Sets out guidance on what to do if a contributor spots a breaking issue in production and explains in which cases to use the rollback, who can perform one and how to do it if it is necessary.

# Tests

No changes to the codes behaviour. All previous tests are passing.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ Include a clear example of the current behaviour with (as much as possible):
 - Screenshots of the behaviour
 - Information on any triage you've attempted
 
+### In case of a breaking issue in Production
+
+Our first priority is to guarantee a stable version in Production (deployed from `main` in our case) at all times, but issues can always come about and in case they do, we have a Rollback Plan!
```
Comment (ShaughnAnderson94) [inline .github/CONTRIBUTING.md:21]: but issues can always come about and in case they do

this bit sounds a bit funny maybe rephrase

**Review by ShaughnAnderson94 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (camelPhonso) [reply .github/CONTRIBUTING.md]: Edited in the latest commit

**Comment by camelPhonso** [comment]
Committed a type declaration change to fix the build error that had creeped into the codebase. Merging this will clear that problem for the other prs

**Review by nichgalzin — APPROVED** [review]
I like how you document. It's real good.
